lib/fennec/evaluator/allocate/request.ex

```diff
@@ -0,0 +1,112 @@
+defmodule Fennec.Evaluator.Allocate.Request do
+  @moduledoc false
+
+  alias Jerboa.Format.Body.Attribute
+  alias Jerboa.Params
+  alias Fennec.TURN
+  @lifetime 10 * 60
+
+  @spec service(Params.t, map, TURN.t) :: {Params.t, TURN.t}
+  def service(x, changes, turn_state) do
```

Comment:
While I see the benefit of having short names in **very local scopes**, I'd at least name them with the first letter of the _long name_, i.e. `p` for `params`, etc.